Snippets/Snippets_6/Transactions.cs

```diff
@@ -32,17 +50,17 @@ public void TransportTransactionsWithScope()
         {
             #region TransactionsWrapHandlersExecutionInATransactionScope
             BusConfiguration busConfiguration = new BusConfiguration();
-            busConfiguration.Transactions()
-                .WrapHandlersExecutionInATransactionScope();
+            busConfiguration.UnitOfWork()
+                .WrapHandlersInATransactionScope();
             #endregion
         }
 
-        public void CustomTransactionTimeout()
+        public void UnitOfWorkWrapHandlersInATransactionScope()
         {
-            #region CustomTransactionTimeout
+            #region UnitOfWorkWrapHandlersInATransactionScope
```

Comment:
The `CustomTransactionTimeout` is there below. It seems I out-smarted the differ by shuffling the code around a little bit too much.

I think we need both `CustomTransactionTimeout` snippet to show in the transport transactions doco and `UnitOfWorkWrapHandlersInATransactionScope` to show in the UoW doco.